resources/conf/log_config.toml

```diff
@@ -29,3 +29,6 @@ logLevel = "INFO"
 enable = false
 logfile = "/tmp/envoy.access.log" # This file will be created inside router container.
 format = "[%START_TIME%] '%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%' %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% '%REQ(X-FORWARDED-FOR)%' '%REQ(USER-AGENT)%' '%REQ(X-REQUEST-ID)%' '%REQ(:AUTHORITY)%' '%UPSTREAM_HOST%'\n"
+
+[debugLogs]
```

Comment:
routerDebugLogs ? wdyt

Reply:
IN the line number 26 we have a comment that states logging configs of Router. After that we have a section called accessLogs. In that case do we can go with.  debugLogs. No need to repeat router again right?

Reply:
+1